src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -476,6 +476,31 @@ public Value get(String what, Value arg)
             return Value.NULL;
         });
 
+        put("absorption",(e, a)->{
+            if(e instanceof PlayerEntity) return new NumericValue(((PlayerEntity) e).getAbsorptionAmount());
+            return Value.NULL;
+        });
+
+        put("xp",(e, a)->{
+            if(e instanceof PlayerEntity) return new NumericValue(((PlayerEntity) e).totalExperience);
+            return Value.NULL;
+        });
+
+        put("xp_level", (e, a)->{
+            if(e instanceof PlayerEntity) return new NumericValue(((PlayerEntity) e).experienceLevel);
+            return Value.NULL;
+        });
+
+        put("xp_points", (e, a)->{//todo add modify implementation to docs.
+            if(e instanceof PlayerEntity) return new NumericValue(((PlayerEntity) e).experienceProgress);
+            return Value.NULL;
+        });
+
+        put("score", (e, a)->{
+            if(e instanceof PlayerEntity) return new NumericValue(((PlayerEntity) e).getScore());
+            return Value.NULL;
+        });
```

Comment:
Style consistency again here. Also shouldn't the TODO be removed?
```suggestion
        put("absorption", (e, a) -> {
            if (e instanceof PlayerEntity) return new NumericValue(((PlayerEntity) e).getAbsorptionAmount());
            return Value.NULL;
        });

        put("xp", (e, a) -> {
            if (e instanceof PlayerEntity) return new NumericValue(((PlayerEntity) e).totalExperience);
            return Value.NULL;
        });

        put("xp_level", (e, a) -> {
            if (e instanceof PlayerEntity) return new NumericValue(((PlayerEntity) e).experienceLevel);
            return Value.NULL;
        });

        put("xp_points", (e, a) -> {//todo add modify implementation to docs.
            if (e instanceof PlayerEntity) return new NumericValue(((PlayerEntity) e).experienceProgress);
            return Value.NULL;
        });

        put("score", (e, a) -> {
            if (e instanceof PlayerEntity) return new NumericValue(((PlayerEntity) e).getScore());
            return Value.NULL;
        });
```